packages/wdio-utils/src/initialiseServices.ts

```diff
@@ -4,14 +4,20 @@ import initialisePlugin from './initialisePlugin'
 
 const log = logger('@wdio/utils:initialiseServices')
 
+type IntialisedService = (
+    [WebdriverIO.ServiceClass | { default: Function }, WebdriverIO.ServiceOption, string] |
+    [WebdriverIO.HookFunctions, Record<string, any>] |
+    [WebdriverIO.ServiceClass, WebdriverIO.ServiceOption]
+)
+
 /**
  * Maps list of services of a config file into a list of actionable objects
  * @param  {Object}    config            config of running session
  * @param  {Object}    caps              capabilities of running session
  * @return {[(Object|Class), Object][]}  list of services with their config objects
  */
-function initialiseServices (services) {
-    const initialisedServices = []
+function initialiseServices (services: [WebdriverIO.ServiceEntry | WebdriverIO.ServiceClass, WebdriverIO.ServiceOption][]): IntialisedService[] {
```

Comment:
Add a Service class:

```suggestion
function initialiseServices (services: Service[]): IntialisedService[] {
```